### DISCUSSION TITLE
Axis tick inside of graph

### DISCUSSION THREAD

**Discussion by kuti-JP** [patrykandpatrick/vico#797] [Questions] [answered]
### Question

Thanks for the great effort put into this library. I was looking for a way to have the ticks drawn inside the graph while having the labels outside, is this feature available? Also is there a release date for Vico 2.0?

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment, marked as answer]
Hi @kuti-JP, this isn't possible yet, but we'll look into adding this shortly.

>Also is there a release date for Vico 2.0?

No, but we're looking into finalizing it and promoting it to beta. Then a stable version will be released relatively soon.

**Reply by kuti-JP** [reply]
Hello again, just wanted to check if this feature is still planned

**Reply by Gowsky** [reply]
Hello, @kuti-JP. Yes, the feature is still planned.
